crates/turborepo-lib/src/commands/config.rs

```diff
@@ -20,6 +20,7 @@ struct ConfigOutput<'a> {
     spaces_id: Option<&'a str>,
     ui: bool,
     package_manager: PackageManager,
+    daemon: Option<bool>,
```

Comment:
Does this need rebasing ro something? This PR is for envMode but diff shows daemon config?

Reply:
yep: it was just stacked :)